portfolio/src/main/webapp/script.js

```diff
@@ -53,27 +52,60 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// Creates a map and adds it to the page with markers on compost-dropoff locations.
+// Global variables that apply across functions
+var map, heatmap;
+var heatmapData = [];
+
+// Creates a map and heatmap layer
 function initMap() {
   // Coordinates centers map  around NYC
   var myLatlng = {lat: 40.763232, lng: -73.951047};
   var mapOptions = {
     center: myLatlng, 
-    zoom: 12,
+    zoom: 14,
```

Comment:
Please make this literal a constant, with a comment describing its purpose.

Reply:
Got it.